mitiq/mitiq_qiskit/tests/test_qiskit_utils.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (C) 2020 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Unit tests for qiskit executors (qiskit_utils.py)."""
+import pytest
+import qiskit
+import numpy as np
+from mitiq.mitiq_qiskit.qiskit_utils import (
+    qs_wvf_sim,
+    qs_wvf_sampling_sim,
+    qs_noisy_sampling_sim,
+)
+
+observable = np.array([[1, 0], [0, 0]])
+two_qubit_observable = np.diag([1, 0, 0, 0])
+shots = 1000
```

Comment:
Could you make these all caps? (e.g., `SHOTS` instead of `shots`) We have this convention for globals.